modular_skyrat/modules/records_on_examine/code/modules/mob/living/record_manifest.dm

```diff
@@ -0,0 +1,87 @@
+/datum/record_manifest
```

Comment:
Is this some copy-paste from a non-modular file or something? Because there's mentions of SKYRAT EDITS and that's kinda silly in a modular file.

Furthermore, does that get updated when people join/leave the round? Because if not, I can see runtimes happening from people going in Cryo.

Reply:
Yeah, most of my code is taken from other things. And yeah, it does work, ive tested it, but I haven't tested for runtimes...